packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -660,14 +684,52 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
               <Stack.Item styles={callGalleryStyles} grow>
                 <Stack verticalFill styles={mediaGalleryContainerStyles}>
                   <Stack.Item styles={notificationsContainerStyles}>
+                    {
+                      /* @conditional-compile-remove(breakout-rooms) */ props.mobileView &&
+                        assignedBreakoutRoom &&
+                        assignedBreakoutRoom.state === 'open' &&
+                        !movingToBreakoutRoomAutomatically && (
```

Comment:
Yes that is set by the Teams meeting organizer.
![image](https://github.com/user-attachments/assets/ea18d152-ba4d-431b-b9df-38f603a470c4). This variable is to prevent showing the 'Join breakout room' button when auto move is on **AND** when the SDK will join the room for us. I use the notifications to determine the second condition. I will rename the variable to something like `autoMoveCurrentlyInEffect`